client/src/Pages/StudentLivePage.js

```diff
@@ -0,0 +1,283 @@
+        };
+  render() {
+    return (
+      <div style={{ "text-align": "center", "margin": "20px 20px" }}>
+        <Card style={{"box-shadow": "0 4px 8px 0 rgba(0,0,0,0.2)", "margin-bottom": "20px"}}>
+          <br></br>
+          <h2><b>Career Fair #1</b></h2>
+          <br></br>
+          <h5><b>Date: </b>Today, October 25, 2020</h5>
+          <br></br>
+          <h5><b>Time: </b>Live Now, 11 PST - 3PM PST</h5>
+          <br></br>
+        </Card>
+        <CardDeck>
+        <Card style={{"box-shadow": "0 4px 8px 0 rgba(0,0,0,0.2)"}}>
+            <div style={{ "width": "200px", "margin": "auto"}}> 
+            <Card.Img variant="top" src={google} height="200" />
+            </div>
+            <Card.Body>
+            <Card.Title>Google</Card.Title>
+            <Card.Text>
+                <br></br>
+                Recruiter: John Doe &nbsp;&nbsp;&nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp; <Button variant="outline-dark" size="sm">In Session</Button>{' '}
+                <br></br>
+                <br></br>
+                Recruiter: John Doe &nbsp;&nbsp;&nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp; <Button variant="outline-secondary" size="sm" href="https://meet.google.com/ezb-yrqf-vsq">Join Now</Button>{' '}
+                <br></br>
+                <br></br>
+                Recruiter: John Doe &nbsp;&nbsp;&nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp; <Button variant="outline-secondary" size="sm" href="https://meet.google.com/ezb-yrqf-vsq">Join Now</Button>{' '}
+                <br></br>
+                <br></br>
+                Recruiter: John Doe &nbsp;&nbsp;&nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp; <Button variant="outline-secondary" size="sm" href="https://meet.google.com/ezb-yrqf-vsq">Join Now</Button>{' '}
+                <br></br>
+                <br></br>
+                Recruiter: John Doe &nbsp;&nbsp;&nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp; <Button variant="outline-secondary" size="sm" href="https://meet.google.com/ezb-yrqf-vsq">Join Now</Button>{' '}
+            </Card.Text>
+            </Card.Body>
+            <Card.Footer>
+            <small className="text-muted">Last updated 3 mins ago</small>
+            </Card.Footer>
+        </Card>
+        <Card style={{"box-shadow": "0 4px 8px 0 rgba(0,0,0,0.2)"}}>
+            <div style={{ "width": "200px", "margin": "auto", "padding-top": "10px"}}> 
+            <Card.Img variant="top" src={microsoft} height="200"/>
+            </div>
+            <Card.Body>
+            <Card.Title>Microsoft</Card.Title>
+            <Card.Text>
+            <br></br>
+                Recruiter: John Doe &nbsp;&nbsp;&nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp; <Button variant="outline-dark" size="sm">In Session</Button>{' '}
+                <br></br>
+                <br></br>
+                Recruiter: John Doe &nbsp;&nbsp;&nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp; <Button variant="outline-secondary" size="sm" href="https://meet.google.com/ezb-yrqf-vsq">Join Now</Button>{' '}
+                <br></br>
```

Comment:
I think the style we have now, we would have just one button per booth as opposed to being able to join a room with any recruiter.
This should be a simple change - I think you just have to reduce the stuff you already have

Reply:
Also the last updated is cool, but if we can replace that with a number of students in queue / estimated wait time - that'd be more appropriate